src/app/[locale]/_component/sessionActiveComp/sessionActiveComp.tsx

```diff
@@ -36,20 +40,11 @@ const SessionExistingComp = ({ title, showArrowBackBtn }: SessionProps): React.R
         <Grid item xs={12} justifySelf={'center'}>
           <Introduction
             title={title}
-            summary={
-              <>
-                <span>
-                  {t.rich('lplogoutpostlogin.activesession', {
-                    deviceModel: 'iPhone 12 Pro',
-                    strong: (chunks) => <strong>{chunks}</strong>,
-                  })}
-                </span>
-              </>
-            }
+            summary={`Al momento hai una sessione attiva sull’app IO, valida fino al ${expirationDate.toLocaleDateString()}. Se hai perso il dispositivo, termina la sessione. Potrai poi accedere nuovamente da qualsiasi altro dispositivo.`}
```

Comment:
Isn't this present in translation files?

Reply:
No it has to be added on ditto and exported the json

Reply:
cc @Ladirico

Reply:
I have not yet entered the texts on ditto because they have not yet been reviewed. We have two possibilities: 
1. enter them now and then edit them 
2. we close this PR and enter them when they are reviewed

Reply:
Got it. So we could close this PR by adding a comment `// FIXME - <JIRA ticket reference for the translation update>` so that we can track this activity.

Reply:
Yes! Now i create a new jira task and next i update the comment of this PR adding the ticket reference

Reply:
[This is](https://pagopa.atlassian.net/browse/IOPID-717) the refence to jira ticket